cmake/CMakeLists.txt

```diff
@@ -1000,14 +1000,6 @@ if(onnxruntime_USE_OPENVINO)
     message(FATAL_ERROR "Unsupported OpenVINO version: ${INTEL_OPENVINO_DIR}")
   endif()
 
-  if (NOT WIN32)
-    if(OPENVINO_VERSION STREQUAL "2020.2")
```

Comment:
was this needed for openvino 2020.2 ? @smkarlap ?

Reply:
nevermind, i see it was moved.

Reply:
It just moved into the onnxruntime_providers.cmake file. 2020.2 compiled just fine with it in the new place (I'm not sure why it's needed for 2020.2)